packages/discovery-react-components/.storybook/tsconfig.json

```diff
@@ -1,6 +1,17 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "isolatedModules": false
+    "isolatedModules": false,
+    "baseUrl": "../src",
+    "paths": {
+      "@rootUtils/*": ["utils/*"],
```

Comment:
can we just do `utils`?

Reply:
I'm open to it. My thought here was that this clearly distinguished it from the `utils` we have inside of some of our components folders.